downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -49,39 +49,39 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     public function main() {
         json & readonly v = {};
         
-        string|error r = v; // compilation error now
-        Employee|string s = v.cloneReadOnly(); // compilation error now
+        string|error r = v; // Compilation error now.
+        Employee|string s = v.cloneReadOnly(); // Compilation error now.
     }
     ```
 
-- Fixed a bug that previously resulted in variables that were initialized with non-isolated expressions being inferred to be `isolated` variables.
+- Fixed a bug that previously resulted in variables that were initialized with non-isolated expressions being inferred as `isolated` variables.
 
     ```ballerina
     int[] config = [];
 
-    // `configs` was previously inferred to be an `isolated` variable
-    // incorrectly. It will no longer be inferred to be an `isolated` 
+    // `configs` was previously inferred as an `isolated` variable
+    // incorrectly. It will no longer be inferred as an `isolated` 
     // variable since `config` is not an isolated expression.
     int[][] configs = [[1, 2], config]; 
 
-    // Since `configs` is not inferred to be an `isolated` variable now,
-    // `getConfig` is not inferred to be an `isolated` function.
+    // Since `configs` is not inferred as an `isolated` variable now,
+    // `getConfig` is not inferred as an `isolated` function.
     function getConfig(int index) returns int[] {
         lock {
             return configs[index].clone();
         }
     }
     ```
 
-- Fixed a bug in dependently-typed function analysis which previously resulted in compilation errors not being logged when the `typedesc` argument is defined using a type definition (`T`) and the return type is a union (`T|t`) where the basic types for `T` and `t` are not disjoint.
+- Fixed a bug (in dependently-typed function analysis) that previously resulted in compilation errors not being logged, when the `typedesc` argument is defined using a type definition (`T`) and the return type is a union (`T|t`) (where the basic types for `T` and `t` are not disjoint).
```

Comment:
Why are we using parentheses for dependently-typed function analysis? It's a key part of the sentence without which the typedesc stuff don't make sense. Similarly for the last bit.

Reply:
```suggestion
- Fixed a bug in dependently-typed function analysis. Previously compilation errors were not logged when the `typedesc` argument is defined using a type definition (`T`) and the return type is a union (`T|t`) in which basic types for `T` and `t` are not disjoint.
```

Reply:
Need to use parentheses here because we need to have the word `that` right after the noun being described to avoid the meaning being distorted. The clause `that previously resulted in compilation errors not being logged` should be placed right after the noun being described by it.

Reply:
Can you check the version I've suggested? I'd rather use that.